app/views/leaderboard.scala.html

```diff
@@ -164,12 +165,100 @@ <h5>@Messages("leaderboard.weekly.detail")</h5>
             </div>
         </div>
     }
+    @if(user && user.get.role.getOrElse("") != "Anonymous" && !UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
```

Comment:
Not a big deal at all, but I'd go with `_.isDefined` over `!_.isEmpty` if possible

Reply:
From what I've found, in scala.html files, isDefined doesn't exist... 😣